src/Apps/Consign/Routes/MarketingLanding/Components/Header.tsx

```diff
@@ -25,6 +23,17 @@ export const Header: React.FC = () => {
     })
   }
 
+  const trackGetInTouchClick = () => {
+    trackEvent({
+      action: "clickedGetInTouch",
+      context_module: "Header",
+      context_page_owner_type: contextPageOwnerType,
+      label: "Get in Touch",
+      user_id: user?.id,
+      user_email: user?.email,
+    })
+  }
```

Comment:
**note:** I think we can keep this as it is now since the screen also contains other hard typed action names, I would keep an eye on this in the future in case we extend it - if so we will need to migrate those actions to cohesion

Reply:
Good point! We could also use the [usePageNameTracking pattern](https://github.com/artsy/force/blob/main/src/Apps/Article/useArticleTracking.ts#L13). I can create a ticket to clean up tracking for the whole MarketingLanding app.

Reply:
great, thanks Ole